### PR TITLE
Removes all remaining uninformative warnings

### LINKED ISSUE #11: Silencing irrelevant warnings
[copied from CUNY-CL/abstractness/issues/101]

I see the following (mostly spurious) warnings on various runs:

* PossibleUserWarning about number of data loaders
* ~UserWarning: deprecation of `nn.functional.sigmoid`~
* UserWarning: dropout expects num_layers > 1
* ~UserWarning: trying to infer the `batch_size` from an ambiguous collection~
* PossibleUserWarning: `max_epochs` was not set.

Users tend to treat warnings as errors, so we may want to suppress these.

### PR DESCRIPTION
**Pull request by kylebgorman**
Closes #11. Tested locally.

### CODE REVIEW

**Comment by Adamits** [comment]
Perfect. Looks good!
